src/deadline/unreal_adaptor/UnrealClient/step_handlers/unreal_render_step_handler.py

```diff
@@ -132,6 +132,11 @@ def executor_failed_callback(executor, pipeline, is_fatal, error):
     def executor_finished_callback(movie_pipeline=None, results=None):
         unreal.log("Render Executor: Rendering is complete")
```

Comment:
Is there reason some still go through the direct call?

Reply:
Its ok, because this callback is needed just for telling the adaptor that unreal client finished the task, no need to wrap with our static log method